.github/actions/argus-builder/build-prep/action.yml

```diff
@@ -21,6 +21,13 @@ inputs:
     description: 'Branch names to run this job on, supports wildcards, comma delimited'
     required: false
     default: ''
+  pr_labels:
+    description: 'Labels attached to an associated PR'
+    required: false
+    default: ''
+  manifest_trigger_label:
```

Comment:
I think this needs to allow multiple values, comma delimited

We don't have anyone with this setup right now but we've previously had people build a monorepo with multiple Argus apps that use independent labels to control stack creation. We have an example monorepo set up to work like that: https://github.com/chanzuckerberg/argus-example-monorepo

Reply:
hmm, this could get confusing. some folks might want it to mean "any of these labels should trigger a manifest update" and others "these labels must all be applied to trigger a manifest update"

what do you think about crossing that bridge when we come to it?

Reply:
I'm good with that, alternatively it could take a regex which would make it support the "any of these labels" case